packages/design-toolkit/src/components/tree/index.tsx

```diff
@@ -28,7 +28,7 @@ import styles from './styles.module.css';
 import type { TreeProps } from './types';
 
 const defaultRenderDropIndicator = (target: DropTarget) => (
-  <DropIndicator target={target} className='border border-highlight-hover' />
```

Comment:
border-highlight-hover is a class from the before tiems, as @switzerb likes to say. so there is no  drop indicator showing on main. i took a stab at what i think it should be
```
.dropIndicator {
    @apply h-xxs bg-(--outline-accent-primary-hover);
}
```

plz lmk if this should be tweaked at all